persistence/src/main/java/edu/unc/lib/dl/services/OperationsMessageSender.java

```diff
@@ -100,6 +100,30 @@ public String sendAddOperation(String userid, Collection<PID> destinations, Coll
         return getMessageId(msg);
     }
 
+    /**
+     * Sends a MarkForDeletion operation message, indicating that an object is being marked for deletion
+     * from the repository
+     *
+     * @param userid id of user who triggered the operation
+     * @param marked object marked for deletion
+     * @return id of operation message
+     */
+    public String sendMarkForDeletionOperation(String userid, PID marked) {
```

Comment:
Given that all the other events take a list of affected pids and mark for deletion is handled as a (potentially) bulk operation in the CDR, it would make sense to have this receive a list as well.  And for it to be called by the MarkForDeletionService instead of the job.

Reply:
fixed!